remote/style.css

```diff
@@ -7,7 +7,7 @@ Description: Remote is a dark, minimal block theme ideal for bloggers. Its defau
 Requires at least: 6.1
 Tested up to: 6.1
 Requires PHP: 5.7
-Version: 1.0.12
+Version: 1.1.0
```

Comment:
I think this should still be `1.0.12`? This is the version listed in the changelog.

Reply:
updated minor version with respect to the changes. 
Reverted it from changes and it is best to bump the version at the time of deployment.